elixir/apps/domain/lib/domain/actors/group/changeset.ex

```diff
@@ -69,39 +69,32 @@ defmodule Domain.Actors.Group.Changeset do
           with: &Actors.Membership.Changeset.for_group(account_id, &1, &2)
         )
 
-      {_data_or_changes, type} when type in [:dynamic, :managed] ->
+      {_data_or_changes, :managed} ->
         changeset
-        |> cast_embed(:membership_rules,
-          with: &Actors.MembershipRule.Changeset.changeset(&1, &2),
-          required: true
-        )
-        |> cast_dynamic_memberships(account_id)
+        |> cast_managed_memberships(account_id)
```

Comment:
Here, we will still roll over all memberships for the `Everyone` group. Since we don't allow changing this group, it should only happen when the group is first created.

Actor/identity updates will hit `update_managed_group_memberships/1` which uses a smarter query.